code.lua

```diff
@@ -3,12 +3,21 @@ local Code = require "code.Code"
 local function printUsage()
   printError("Usage:")
   printError("  " .. arg[0] .. " <filename>")
+  printError("  " .. arg[0] .. " --update")
+end
+
+local function update()
+  print("Please wait...")
+  shell.run("wget run https://raw.githubusercontent.com/Possseidon/cc-code/main/code/update.lua")
 end
 
 local args = { ... }
 if #args ~= 1 then
   printUsage()
   return
+elseif args[1] == "--update"
```

Comment:
```suggestion
elseif args[1] == "--update" then
```

You're missing a `then`.

Also I probably want proper argument parsing at some point, but for now this will have to do ^^